pkg/reconciler/tenancy/bootstrap/bootstrap_reconcile.go

```diff
@@ -47,7 +49,13 @@ func (c *controller) reconcile(ctx context.Context, workspace *tenancyv1alpha1.C
 	klog.Infof("Bootstrapping resources for org workspace %s, logical cluster %s", workspace.Name, wsClusterName)
 	bootstrapCtx, cancel := context.WithDeadline(ctx, time.Now().Add(time.Second*30)) // to not block the controller
 	defer cancel()
-	if err := c.bootstrap(bootstrapCtx, c.crdClient.Cluster(wsClusterName).Discovery(), c.dynamicClient.Cluster(wsClusterName), c.kcpClient.Cluster(wsClusterName)); err != nil {
+
+	clusterConfig := kcpclienthelper.ConfigWithCluster(c.config, wsClusterName)
```

Comment:
this is a miss-namer. We usually call something "clusterSomething" if it is NOT attached to a logical cluster. This one is more like `wsConfig`.